site/src/docs/getting-started/tutorials/migrate-to-3.0.0.mdx

```diff
@@ -0,0 +1,107 @@
+---
+slug: '/getting-started/tutorials/migrate-to-3.0.0'
+title: 'Migrate to 3.0.0'
+navTitle: 'Migrate to 3.0.0'
+---
+
+import ExternalLink from '../../../components/ExternalLink';
+import LeadParagraph from '../../../components/LeadParagraph';
+import InternalLink from '../../../components/InternalLink';
+
+# Migrate to version 3.0.0
+This guide helps you to start using HDS version 3.0.0. 
+
+## Breaking changes
+This major release contains some breaking changes. Pay attention to these when you update your project to use the new version.
+
+### Navigation replaced with Header
+Old navigation was removed as previously informed. The aim is to provide more flexible navigation solution that supports also larger sites and is consistent with hel.fi-styles.
+This new navigation is not updated automatically, you need to switch to it manually as the name of the component is different.
+
+If you wish to make it as similar to previous navigation, you only need ActionBar-subcomponent.
+
+The navigation was rewritten to fix bugs, make it more modular, provide support for larger sites and unify the styles.
+
+- <InternalLink size="M" href="/components/header">Header component</InternalLink>
+- <InternalLink size="M" href="/patterns/navigation/building-navigation/">Navigation pattern</InternalLink>
+
+### New footer
+Footer has been refactored and the new version is more modular. The new component also is named footer, so check that needed properties are set correctly.
+
+- <InternalLink size="M" href="/components/footer">Footer component</InternalLink>
+
+### Deprecated rectangular tags and statuslabels
+Rectangular tags and status labels have been deprecated and removed. Use rounded version instead.
+This helps to better separate buttons from tags.
+- <InternalLink size="M" href="/components/tag">Tag component</InternalLink>
+- <InternalLink size="M" href="/components/status-label">Status label component</InternalLink>
+
+
+### External link icon position adjusted
+The position of the external link is adjusted to better match common practices. This moves the icon bit more up, which can affect the line height.
+
+### Properties changes
+- LinkBox property name 'withBorder' has been changed to 'border'.
+- Koro functionality flipHorizontal has been renamed to flipVertical to better reflect the actual behaviour.
+- Koro shapes had mixed up names in English. Wave was changed to Vibration and Storm to Wave.
+
+## Other important updates
+
+### Focus color
+New focus color token has been added. Currently the default focus color is still Coat of Arms, but token enables changing the color according to the needs.
+Components are updated to use the new token instead of hardcoded value.
```

Comment:
This has already been released in a minor release but did you want to praise it here also?

Reply:
Same here, this kind of slipped without praises earlier, I'll improve the text but give some praises still.